openpype/hosts/maya/api/lib.py

```diff
@@ -3982,6 +3982,6 @@ def create_rig_animation_instance(nodes, context, namespace, log=None):
             creator_plugin,
             name=namespace,
             asset=asset,
-            options={"useSelection": True},
+            options=options.update({"useSelection": True}),
```

Comment:
Update doesn't return the dict itself. This wouldn't work. Also the mutable `options` dict I believe is re-used for multiselection loading so I suspect this shouldn't be done like this.

````python
create_options = options.copy()
create_options["useSelection"] = True
```
And potentially a `copy.deepcopy` might be even safer.

Reply:
Thank you, @BigRoy. Finally, we do not need to change options here.

Reply:
I'd argue the opposite actually. We still want to make sure `useSelection: True` is passed as an option even if the rig loader was called with it False.